modules/deploy/partials/linux/install-fips.adoc

```diff
@@ -0,0 +1,28 @@
+= Install Redpanda for FIPS compliance
+
+To run Redpanda in FIPS compliance mode, you must install the following additional packages after installing Redpanda:
+
+`redpanda-fips`
+
+- Contains the OpenSSL FIPS-approved module and scripts required to set up and run Redpanda in FIPS-compliance mode.
+- Depends upon the successful installation of the `redpanda` package.
```

Comment:
Should this be a step? Since we already say "after installing Redpanda" right before the bullet points

Reply:
It's unclear to me whehter or not you must first install the redpanda package, and then the fips packages, OR, if those packages get install as part of the redpanda install. Think I'll await clarification from @michael-redpanda or @deniscoady . This file is not appearing in the generated output for the PR, but it does in my local build.

Reply:
The `redpanda-fips` package is separate from `redpanda`.  The `redpanda-fips` package depends on `redpanda` already being installed.

So one could do `sudo apt install -y redpanda-fips` and that would grab `redpanda` automagically.

For FIPS installs I think being pedantic and explicit are a good thing.  So something like:

`sudo apt install -y redpanda-rpk-fips redpanda-tuners redpanda redpanda-fips` 

I think is acceptable.